docs/platform/reference/eol-for-major-versions.rst

```diff
@@ -0,0 +1,167 @@
+as the upstream open source project's EOL .
+
+.. container:: intercom-interblocks-table-container
+
+   =========== ============= ==================
+   **Version** **Aiven EOL** **Upstream EOL**
+   2.x         2020-10-30    2018-02-28
+   5.x         2020-10-30    2019-03-11
+   6.x         2020-11-20    2020-11-20
+   7.x         2022-03-23    2022-05-11 (v7.10)
+   =========== ============= ==================
+
+Aiven for PostgreSQL®
+---------------------
+
+Aiven for PostgreSQL® major versions will reach EOL on the same date as
+the upstream open source project's EOL .
+
+.. container:: intercom-interblocks-table-container
+
+   +-------------+---------------+------------------+------------------+
+   | **Version** | **Aiven EOL** | **Availability   | **Availability   |
+   |             |               | end on Aiven     | start on Aiven   |
+   |             |               | Platform**       | Platform**       |
+   +-------------+---------------+------------------+------------------+
+   | 9.5         | 2021-04-15    | 2021-01-26       | 2015-12-22       |
+   +-------------+---------------+------------------+------------------+
+   | 9.6         | 2021-11-11    | 2021-05-11       | 2016-09-29       |
+   +-------------+---------------+------------------+------------------+
+   | 10          | 2022-11-10    | 2022-05-10       | 2017-01-14       |
+   +-------------+---------------+------------------+------------------+
+   | 11          | 2023-11-09    | 2023-05-09       | 2017-03-06       |
+   +-------------+---------------+------------------+------------------+
+   | 12          | 2024-11-14    | 2024-05-14       | 2019-11-18       |
+   +-------------+---------------+------------------+------------------+
+   | 13          | 2025-11-13    | 2025-05-13       | 2021-02-15       |
+   +-------------+---------------+------------------+------------------+
+   | 14          | 2026-11-12    | 2026-05-12       | 2021-11-11       |
+   +-------------+---------------+------------------+------------------+
+
+.. _aiven-for-kafka:
+
+Aiven for Apache Kafka®
+-----------------------
+
+Starting with v2.5, Aiven for Kafka® ``major.minor`` version will reach
+EOL one year after it's made available on Aiven platform.
+
+.. container:: intercom-interblocks-table-container
+
+   =========== ============= ========================================
+   **Version** **Aiven EOL** **Availability start on Aiven Platform**
+   1.0.x       2021-02-01    2017-11-01
+   1.1.x       2021-02-01    2018-07-31
+   2.0.x       2021-02-01    2018-07-30
+   2.1.x       2021-02-01    2018-12-04
+   2.2.x       2021-02-01    2019-04-19
+   2.3.x       2021-08-13    2019-09-05
+   2.4.x       2021-08-13    2019-10-21
+   2.5.x       2021-08-13    2020-05-05
+   2.6.x       2021-08-13    2020-08-13
+   2.7.x       2022-01-24    2021-01-21
+   2.8.x       2022-06-02    2021-04-26
+   3.0         2022-11-22    2021-11-22
+   3.1         2023-02-14    2022-02-14
```

Comment:
3.2 now needs adding

Reply:
@sbijjal do we have the dates for Kafka 3.2 EOL?

Reply:
It's in https://help.aiven.io/en/articles/4318172-eol-for-major-versions-of-aiven-services

Reply:
Thanks @ivanyu!
@sbijjal could you confirm about the EOL dates for 3.0?